src/components/Identity/index.js

```diff
@@ -29,11 +36,50 @@ const createIdentity = ({ config, logger, cookie }) => {
   let optIn;
   let responseRequested = false;
   let deferredForEcid;
+  let network;
+  let lifecycle;
+  const customerIDs = {};
+  const getCustomerIDs = () => customerIDs;
+
+  const makeServerCall = event => {
+    const payload = network.createPayload();
+    payload.addEvent(event);
+    const responsePromise = Promise.resolve()
+      .then(() => {
+        return lifecycle.onBeforeDataCollection(payload, responsePromise);
+      })
+      .then(() => {
+        return network.sendRequest(payload, payload.expectsResponse);
+      });
+
+    return responsePromise;
+  };
+
+  const setCustomerIDs = options => {
+    const event = createEvent();
+    validateCustomerIDs(options.customerIDs);
+    Object.keys(options.customerIDs).forEach(key => {
+      customerIDs[key] = options.customerIDs[key];
+    });
+    const normalizedCustomerIDs = normalizeCustomerIDs(customerIDs);
+    const customerIDsHash = createHashFromString(
+      serializeCustomerIDs(normalizedCustomerIDs)
+    );
+    const hasSynced = customerIDsHash === cookie.get("CIDH");
```

Comment:
Use the constant you created for CIDH (and on line 72)

Reply:
Done 👍